src/evaluators/loader.py

```diff
@@ -17,3 +28,30 @@ def load_infmax_model(config: dict[str, Any], random_seed: int, nb_seeds: int) -
         return load_model({"model": config})
     else:
         return lambda x: "dupa"  # TODO: add here GT loader
+
+
+def if_stochastic(infmax_model: Callable) -> bool:
```

Comment:
[1]
Why `Callable`? Seems that it works for any object

Reply:
I wanted to narrow down that it shall be any object with `__call__` implemented